site/en/adminGuide/tls.md

```diff
@@ -0,0 +1,456 @@
+crypto_device	= builtin		# OpenSSL engine to use for signing
+signer_cert	= $dir/tsacert.pem 	# The TSA signing certificate
+					# (optional)
+certs		= $dir/cacert.pem	# Certificate chain to include in reply
+					# (optional)
+signer_key	= $dir/private/tsakey.pem # The TSA private key (optional)
+
+default_policy	= tsa_policy1		# Policy if request did not specify it
+					# (optional)
+other_policies	= tsa_policy2, tsa_policy3	# acceptable policies (optional)
+digests		= md5, sha1		# Acceptable message digests (mandatory)
+accuracy	= secs:1, millisecs:500, microsecs:100	# (optional)
+clock_precision_digits  = 0	# number of digits after dot. (optional)
+ordering		= yes	# Is ordering defined for timestamps?
+				# (optional, default: no)
+tsa_name		= yes	# Must the TSA name be included in the reply?
+				# (optional, default: no)
+ess_cert_id_chain	= no	# Must the ESS cert id chain be included?
+				# (optional, default: no)
+```
+
+</details>
+
+<details><summary><code>gen.sh</code></summary>
+
+```shell
+#!/usr/bin/env sh
+# your variables
+Country="CN"
+State="Shanghai"
+Location="Shanghai"
+Organization="milvus"
+Organizational="milvus"
+CommonName="localhost"
+
+echo "generate ca.key"
+openssl genrsa -out ca.key 2048
+
+echo "generate ca.pem"
+openssl req -new -x509 -key ca.key -out ca.pem -days 3650 -subj "/C=$Country/ST=$State/L=$Location/O=$Organization/OU=$Organizational/CN=$CommonName"
+
+echo "generate server SAN certificate"
+openssl genpkey -algorithm RSA -out server.key
+openssl req -new -nodes -key server.key -out server5.csr -days 3650 -subj "/C=$Country/O=$Organization/OU=$Organizational/CN=$CommonName" -config ./openssl.cnf -extensions v3_req
+openssl x509 -req -days 3650 -in server.csr -out server.pem -CA ca.pem -CAkey ca.key -CAcreateserial -extfile ./openssl.cnf -extensions v3_req
+
+echo "generate client SAN certificate"
+openssl genpkey -algorithm RSA -out client.key
+openssl req -new -nodes -key client.key -out client.csr -days 3650 -subj "/C=$Country/O=$Organization/OU=$Organizational/CN=$CommonName" -config ./openssl.cnf -extensions v3_req
+openssl x509 -req -days 3650 -in client.csr -out client.pem -CA ca.pem -CAkey ca.key -CAcreateserial -extfile ./openssl.cnf -extensions v3_req
+
+```
+</details>
+
+### Create certificate
+
+Run the `gen.sh` file to create the certificate.
+
+```
+chmod +x gen.sh
+./gen.sh
+```
+
+The following nine files will be created: `ca.key`, `ca.pem`, `ca.srl`, `server.key`, `server.pem`, `server.csr`, `client.key`, `client.pem`, `client.csr`.
+
+## Modify configurations
+
+Set `tlsEnabled` to `true` and configure the file path in `config/milvus.yaml`.
+
+The `server.pem`, `server.key`, and `ca.pem` files for the server and the `client.pem`, `client.key`, and `ca.pem` files for the client need to be configured.
+
+```
+tls:
+  serverPemPath: configs/cert/server.pem
+  serverKeyPath: configs/cert/server.key
+  caPemPath: configs/cert/ca.pem
+
+common:
+  security:
+    tlsEnabled: true
+ ```
+ 
```

Comment:
Are we missing next step(s) in this doc, such as generating key and certificate for each node?

> Ref: https://docs.yugabyte.com/preview/secure/tls-encryption/server-certificates/#generate-key-and-certificate-files-for-each-node

Reply:
BinBin believes that all necessary information are covered. Added one last section about adding the tls-related parameters when connecting to the Milvus server.